### ISSUE TITLE
Accessibility: Some missing button labels in Talk for iOS

### ISSUE THREAD

**Issue by MarcoZehe** [nextcloud/spreed#3132]
## Steps to reproduce
1. Install NextCloud Talk from app store, currently v8.1.
2. Log into a server.
3. Start VoiceOver and explore the UI.

### Expected behaviour
All buttons should be labeled.

### Actual behaviour
Some buttons like the Settings button, the Audio and Video Call buttons in a conversation, are not labeled. VoiceOver may guess the label from the image name or its build-in text recognition on images, but that does not always work. Dedicated AccessibilityLabel properties on image buttons are better.

NextCloud for iOS on iPhone, version 8.1.0.

CC @nextcloud/accessibility
